Tools/CISupport/ews-build/results_db.py

```diff
@@ -103,6 +168,28 @@ def is_test_pre_existing_failure(cls, test, commit=None, configuration=None, sui
         }
         defer.returnValue(output)
 
+    @classmethod
+    @defer.inlineCallbacks
+    def does_result_match(cls, test, result_type=None, commit=None, configuration=None, suite=None, default=None):
```

Comment:
Where is this actually used?

Reply:
This will be used in `FindUnexpectedSaferCPPResults` (it's where I tested this function in the build I linked). I'll specify the exact spot once I update the PR for those changes.